.github/workflows/release.yml

```diff
@@ -136,33 +133,36 @@ jobs:
         uses: softprops/action-gh-release@v1
         with:
           tag_name: ${{ github.event.inputs.release_version }}
-          name: Release ${{ github.event.inputs.release_version }}
-          body: |
-            Automated release for version ${{ github.event.inputs.release_version }}.
-              - Published to Maven Central
-              - Published to npm
-              - Built runnable JARs included
-          
-            [Full Changelog](https://github.com/vanillabp/business-cockpit/compare/${{ env.LAST_RELEASED_VERSION }}...${{ github.event.inputs.release_version }})
+          name: v${{ github.event.inputs.release_version }}
+          generate_release_notes: true
+          append_body: |
+            
+            **Full Changelog:** [`${{ env.LAST_RELEASED_VERSION }}...${{ github.event.inputs.release_version }}`](https://github.com/vanillabp/business-cockpit/compare/${{ env.LAST_RELEASED_VERSION }}...${{ github.event.inputs.release_version }})
           files: |
             **/*-runnable.jar
           draft: true
 
+      - name: Calculate next snapshot version
+        run: |
+          NEXT_SNAPSHOT==${LAST_RELEASED_VERSION}-SNAPSHOT
```

Comment:
Shouldn't the last released version get incremented for the next snapshot version?